.github/README.md

```diff
@@ -12,11 +12,21 @@ Additionally, the OpenTelemetry Contrib collector has also been changed to the [
 
 ## Docker compose
 
+### Elasticsearch exporter (default)
 1. Start a free trial on [Elastic Cloud](https://cloud.elastic.co/) and copy the `Elasticsearch endpoint` and the `API Key` from the `Help -> Connection details` drop down instructions in your Kibana. These variables will be used by the [elasticsearch exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticsearchexporter#elasticsearch-exporter) to authenticate and transmit data to your Elasticsearch instance.
 2. Open the file `src/otel-collector/otelcol-elastic-config.yaml` in an editor and replace all occurrences the following two placeholders:
    - `YOUR_ELASTICSEARCH_ENDPOINT`: your Elasticsearch endpoint (*with* `https://` prefix example: `https://1234567.us-west2.gcp.elastic-cloud.com:443`).
    - `YOUR_ELASTICSEARCH_API_KEY`: your Elasticsearch API Key
-3. Start the demo with the following command from the repository's root directory:
+
+### Managed Ingest Endpoint
+1. Sign up for a free trial on [Elastic Cloud](https://cloud.elastic.co/) and start an Elastic Cloud Serverless Observability type project. Select Application and then OpenTelemetry.
+2. Copy the OTEL_EXPORTER_OTLP_ENDPOINT URL and replace `.apm` with `.ingest`.
```

Comment:
The endpoint with `.apm` is pointing to the OTLP endpoint of the APM Server, and the `.ingest` is pointing to the OTel collector, right?

Should we maybe just add a short note about that here? Otherwise it's a bit confusing and people may not know what's going on here.

_We have the same below for the Kubernetes part._

Reply:
Agreed offline to keep these instructions and modify the README later on